frontend/data/EvalsStore.ts

```diff
@@ -0,0 +1,14 @@
+import { create } from 'zustand';
+import {
+  createEvalSlice,
+  type EvalSlice,
+} from '@frontend/data/slices/EvalSlice';
+import {
+  createSharedSlice,
+  type SharedSlice,
+} from '@frontend/data/slices/SharedSlice';
+
+export const EvalsStore = create<EvalSlice & SharedSlice>()((...params) => ({
+  ...createEvalSlice(...params),
+  ...createSharedSlice(...params),
+}));
```

Comment:
We should add other middlewares here as well (persist, debug, anything custom that we want etc.)
They can differ per store based on the needs of the page (tasks might want to persist data whilst evals doesn't)